docs/guide/persistence_cache.md

```diff
@@ -157,7 +179,11 @@ Platform.sh Professional and lower versions offer Redis in single instance mode
 See [Memcached Cache Adapter in Symfony documentation](https://symfony.com/doc/3.4/components/cache/adapters/memcached_adapter.html#configure-the-connection)
 for information on how to configure Memcached.
 
-Example:
+Out of the box in `app/config/cache_pool/cache.memcached.yml` you'll find a default example that can be used.
+
+For eZ Platform Cloud/Platform.sh; This is automatically configured in `app/config/env/platformsh.php` if you have enabled memcached as `cache` platform.sh service.
+
+For anything else you can enabled it with environment variables detected by `app/config/env/generic.php`; If you set the following environment variables `export CACHE_POOL="cache.memcached" CACHE_DSN="user:pass@localhost?weight=33"`, it will result in config like this:
```

Comment:
```suggestion
For anything else, you can enable it with environment variables detected automatically by `app/config/env/generic.php`. For instance, if you set the following environment variables `export CACHE_POOL="cache.memcached" CACHE_DSN="user:pass@localhost?weight=33" CACHE_NAMESPACE="ez"`, it will result in config like this:
```